guides/common/modules/proc_adding-proxmox-details-to-a-compute-profile.adoc

```diff
@@ -12,34 +12,15 @@ Note that this procedure assumes you are using KVM/qemu-based virtualization.
+In Proxmox, you can choose between *LXC container* (Linux containers) and *KVM/Qemu server* (kernel-based virtual machines).
 LXC focuses more on isolating applications and reusing the host kernel.
 KVM runs its own kernel and lets you boot various guest operating systems.
+. Click on the *General* tab.
 . In the *Node* list, select the Proxmox node you want to deploy your hosts to.
 This makes sense if you run a Proxmox cluster.
-. If you want to automatically start virtual machines after creation, select the *Start after creation?* checkbox.
-You can overwrite this for individual hosts when you create them.
 . In the *Image* list, select an image that is available on your Proxmox compute resource.
 . In the *Pool* list, select a resource allocation pool.
-// FIXME: this procedure currently ignores advanced options "Main options", "CPU", "memory", and "OS"
-. Optional: You can select the *Main options*, *CPU*, *Memory*, and *OS* checkboxes to configure the compute profile in more detail.
-. In the *Identifier* field, enter the name of the network interface of your host.
-The default value is `net0`.
-. From the *Card* list, select the type of network interface card of your host.
-. Optional: From the *Bridge* list, select a bridge to assign the host to it.
-. Optional: In the *VLAN tag* field, you can assign your host to a VLAN.
-The *VLAN tag* field lets you input the corresponding VLAN tag which your Proxmox node is part of.
-. Optional: In the *Rate limit* field, you can limit the network traffic of hosts.
-. Optional: In the *Multiqueue* field, you can set multiple virtual CPUs to process network packets.
-. Optional: If you want to use the integrated firewall in Proxmox, select the *Firewall* checkbox.
-. Optional: If you want to disconnect your host from the network, select the *Disconnect* checkbox.
-. Optional: Click *Add Interface* to add another network interface.
-// FIXME: this procedure currently ignores "Add Hard Disk", "Add CD-ROM", and "Add Cloud-init"
-. From the *Storage* list, select the name of the storage pool within Proxmox.
-. In the *Controller* field, select the type of the disk controller of your host in Proxmox.
-{Team} recommends to use either _SCSI_ or _VirtIO Block_.
-. In the *Device* field, enter the number of the disk.
-. In the *Cache* list, select the type of disk cache you want to use on your host in Proxmox.
-{Team} does not recommend using caching within Proxmox.
-. In the *Size (GB)* field, enter the size of the disk.
+. Optional: Click on *Advanced Options*, *Hardware*, *Network Interfaces*, or *Storage* and follow the {ProjectWebUI} to configure the compute profile.
+. On the *Storage* tab, {Team} recommends to use either _SCSI_ or _VirtIO Block_ in the *Controller* field as the type of the disk controller of your host in Proxmox.
+. On the *Storage* tab, {Team} does not recommend to use caching within Proxmox.
```

Comment:
I think this requires an explanation. You might want to mention this in the intro to the procedure.